pep-0619.rst

```diff
@@ -76,6 +76,12 @@ Actual:
   installers)
 - 3.10.12: Tuesday, 2023-06-06
 
+Source-only security fix releases
+---------------------------------
+
+Provided irregularly on an "as-needed" basis until October 2026.
+
+- 3.10.13: Thursday, 2023-08-24
```

Comment:
Should 3.10.12 be in this new section?

Reply:
Good spot, yes it should. Updated.

re: https://www.python.org/downloads/release/python-31012/